src/fides/api/models/privacy_experience.py

```diff
@@ -28,32 +28,32 @@ class ComponentType(Enum):
     privacy_center = "privacy_center"
 
 
-class DeliveryMechanism(Enum):
+class BannerEnabled(Enum):
     """
-    The delivery mechanism - not formalized in the db
+    Whether the banner should display
     """
 
-    banner = "banner"
-    link = "link"
+    always_enabled = "always_enabled"
+    enabled_where_required = "enabled_where_required"  # If the user's region has at least one opt-in or notice-only notice
+    always_disabled = "always_disabled"
 
 
 class ExperienceConfigBase:
     """Base schema to share common experience config."""
 
-    acknowledgement_button_label = Column(String)
-    banner_title = Column(String)
-    banner_description = Column(String)
```

Comment:
Seems like there was a simplification in the UI that we can just define a title and a description, and they can both be used for a banner title and an overlay title, so I'm removing the duplicate set of fields.